packages/graphql/src/schema/validation/directives.ts

```diff
@@ -176,7 +176,7 @@ export const nodeDirective = new GraphQLDirective({
             type: new GraphQLList(new GraphQLNonNull(GraphQLString)),
         },
         plural: {
-            description: "Defines a custom plural for the Node API",
+            description: "Allows for the specification of the plural of the type name if the type name is non-English.",
```

Comment:
Great on changing the old description, which is awful in hindsight.
**nitpick** I think there are other cases in which plural directive may be useful, other than non-English typenames.

For instance "Tech" -> "Teches".

```suggestion
            description: "Allows for the specification of the plural of the type name.",
```